app-backup/optar/optar-1.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Codec for encoding data on paper"
+HOMEPAGE="https://github.com/colindean/optar"
+SRC_URI="https://github.com/colindean/${PN}/archive/${PV}-colindean.tar.gz -> ${P}.tar.gz"
```

Comment:
Please use the same name for distfile as is used for the directory inside it, i.e. `${P}-colindean`. A common practice is to create a helper `MY_P=${P}-colindean` variable to avoid repeating that twice but it's up to you.